README.md

```diff
@@ -1,2 +1,44 @@
-# the-next-web-clone
-The Odin Project → HTML &amp; CSS Course → Assignment #06
+# TNW-clone
+
+This page is a clone of a TNW page made for the Milestone [6] project of Microverse &amp; Odin Project.
+
+![screenshot](screenshot.png)
+
+This project consists of building an HTML document that matches the appearance of The Next Web page.
+
+## Built With
+
+- HTML, css
+- BIM
+- 7-1 Pattern
+- No frameworks
+
+## Live Demo
+
+[Live Demo Link]https://rawcdn.githack.com/MohameDawood/the-next-web-clone/3d107e8b88aea6abf23959ff01b229ad1d5c5d16/index.html)
```

Comment:
you forgot to open the parentheses for the link please do that and request another code review

Reply:
Done